api/aether-2.0.0-openapi3.yaml

```diff
@@ -1443,6 +1523,23 @@ paths:
         "201":
           description: created
       summary: POST /enterprises/enterprise
+  /aether/v2.0.0/{target}/enterprises/enterprise:
```

Comment:
this might be for later when we discuss overall api design but can this be `/aether/v2.0.0/{target}/enterprises` now or would that impact too much?

Reply:
This already exists on line 1458 above - that is for the container `/enterprises`
This one here is for the list items `/enterprises/enterprise`
The 2 will give the same result because the there is no other list, leaf or container in the `/enterprises`, but if there was there would be a difference between the 2